cypress/models/employee/employee.ts

```diff
@@ -0,0 +1,8 @@
+export interface Employee {
+  firstName: string;
+  lastName: string;
+  email: string;
+  editFirstName: string;
```

Comment:
Can we use better naming for `editFirstName` and remaining params? What is the context here? What end-user expect and/or should do here?